src/api.py

```diff
@@ -22,18 +32,30 @@ class PromptGenerationPlugin(Tagger):
     """
 
     class PromptGenerationPluginConfig(Config):
-        openai_api_key: str = Field("", description="An openAI API key to use. If left default, will use Steamship's API key.")
-        max_words: int = Field(description="The maximum number of words to generate per request")
-        model: Optional[str] = Field("text-davinci-003", description="The OpenAI model to use.  Can be a pre-existing fine-tuned model.")
-        temperature: Optional[float] = Field(0.4, description="Controls randomness. Lower values produce higher likelihood / more predictable results; higher values produce more variety. Values between 0-1.")
-        top_p: Optional[int] = Field(1, description="Controls the nucleus sampling, where the model considers the results of the tokens with top_p probability mass. Values between 0-1.")
-        n_completions: Optional[int] = Field(1, description="How many completions to generate for each prompt.")
-        echo: Optional[bool] = Field(False, description= "Echo back the prompt in addition to the completion")
-        stop: Optional[str] = Field("", description="Up to 4 sequences where the API will stop generating further tokens. The returned text will not contain the stop sequence. Value is comma separated string version of the sequence.")
-        presence_penalty: Optional[int] = Field(0, description="Control how likely the model will reuse words. Positive values penalize new tokens based on whether they appear in the text so far, increasing the model's likelihood to talk about new topics. Number between -2.0 and 2.0.")
-        frequency_penalty: Optional[int] = Field(0, description="Control how likely the model will reuse words. Positive values penalize new tokens based on their existing frequency in the text so far, decreasing the model's likelihood to repeat the same line verbatim. Number between -2.0 and 2.0.")
-        best_of: Optional[int] = Field(1, description="Generates best_of completions server-side and returns the \"best\" (the one with the highest log probability per token).")
-        moderate_output: bool = Field(True, description="Pass the generated output back through OpenAI's moderation endpoint and throw an exception if flagged.")
+        openai_api_key: str = Field("",
+                                    description="An openAI API key to use. If left default, will use Steamship's API key.")
+        max_tokens: int = Field(description="The maximum number of tokens to generate per request")
```

Comment:
Given that max_words was a required parameter, and basically none of our package code requests specific versions of the plugins, changing the name of the param would break all of the existing packages that use this plugin.  We'd have to update *everything* simultaneously.  Do you feel like these renames are worth it right now?

If we want to align on OpenAI's parameter names, and we're moving away from this being 'prompt-gen-default' to 'GPT-3', I think we should version the plugins in all our existing code first, then do this renaming, so that we can do it without having to crazily scramble.